scripts/dev/make_multiv_table_fields.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Comment:
Here is a small snippet that could be useful if we ever want to change field numbers for that object again.